src/components/ProjectsList/ProjectList.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { List, LinkItem } from "../List/List";
+import "./ProjectList.css";
+import UserForm from "../userForm/UserForm";
+const projects = [
+  "project 1",
+  "project 2",
+  "project 3",
+  "project 4",
+  "project 5",
+];
```

Comment:
Can you amend this dummy data to use project objects and amend rendering to use display different info? E.g.
`const projects = [
  {_id: 234iuhowef9803rh, title: "Big Ben", .......},
  {..........},
];`
That way there will be less to change in the future